src/validation/validationHealthcareProfessional.ts

```diff
@@ -368,3 +368,70 @@ export function validateSpokenLanguages(
         })
     }
 }
+
+export function validateUpdateProfessionalInput(input: Partial<gqlTypes.UpdateHealthcareProfessionalInput>)
```

Comment:
@Ghezz9 NIT: it can be in separate pull request but some sort of unit tests would be nice.